docs/cloud/private-link.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Private Link
+description: Private Link enables lakeFS Cloud to interact with your infrastructure using private networking.
+parent: lakeFS Cloud
+nav_order: 30
+has_children: false
+redirect_from: "/cloud.html"
+---
+
+# Private-Link
```

Comment:
Per other comment I don't mind what it's called but make sure the term is consistent throughout
```suggestion
# Private Link
{: .no_toc }
```